src/main/java/net/minestom/server/entity/Entity.java

```diff
@@ -1452,6 +1455,122 @@ public void takeKnockback(final float strength, final double x, final double z)
         }
     }
 
+    public Collection<Entity> getNearbyEntities(double range) {
```

Comment:
Should probably be in `Instance`

Reply:
Did it and also added synchronization over entities lock for the whole cycle